crates/core/src/types/type.rs

```diff
@@ -209,40 +209,34 @@ impl From<&str> for Type {
     }
 }
 
+macro_rules! type_ids {
+    ($($type:ty),*) => {
+        type_ids![$($type,)*]
+    };
+    ($($type:ty,)*) => {
```

Comment:
How does this work, isn't it the same pattern as the first one?

Reply:
Trailing comma vs non trailing comma. This syntax allows both :)

Reply:
ooooh 😅